extensions/amp-pinterest/0.1/amp-pinterest.js

```diff
@@ -350,7 +350,7 @@ class AmpPinterest extends AMP.BaseElement {
         '?pin_ids=' + pinId +
         '&sub=www&base_scheme=https';
       return call(query).then(r => {
```

Comment:
we can probably just do:

```js
return call(query).then(renderPin.bind(this));
```

which would get rid of the `self` param as well.